pacientes/precadastropaciente.jh

```diff
@@ -0,0 +1,30 @@
+entity PreCadastroPaciente {
+
+    nome String required maxlength(40)
+    nomeSocial String required maxlength(20)
+    nomeDaMae String required maxlength(40)
+    dataDeNascimento LocalDate required
+    cartaoSus String required maxlength(15)
+    status Boolean
```

Comment:
rever nome do campo, por ser `Boolean`, deveria ter por nome `ativo` se for para marcar o registro como ativo ou inativo por exemplo.